scripts/inf.sh

```
@@ -0,0 +1,59 @@
+source ~/.bashrc
+
+conda activate thermoMPNN
+
+export CUDA_VISIBLE_DEVICES=7
+echo "Running script on GPU $CUDA_VISIBLE_DEVICES"
+
+cd /home/hdieckhaus/scripts/ThermoMPNN/proteinmpnn/
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)